### DISCUSSION TITLE
Translate recipe to another language when importing recipe

### DISCUSSION THREAD

**Discussion by ZombieSmurf** [mealie-recipes/mealie#3918] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

We are a Danish household, using the Daniash names and the metric system for the ingredients. When I need to add a non Danish recipe, I need to parse it through ChatGPT for translation (or just translate it myself, but Chaaat is just much faster) and copy the translated text in.
It would be pretty helpfull if Mealie could do it automatic.

### Please provide a concise description of the feature that would resolve your issue.

An option for translating the recipe and ingredient measures to a given language using openai, when importing.

It's in the same ball park as Feature Req. 3011 (https://github.com/mealie-recipes/mealie/discussions/3011), but not entirely the same. When ChatGPT does the translation, it's far from perfect. Some phrases are not translated in a meningfull way, measurements might be a bit too litteral (like 1 1/3 cup of sugar = 0,308 liters) and needs to me manually corrected before saving the recipe.

### Please consider and list out some caveats or tradeoffs made in your design decision

It might need to be something that should be manually enabled for each import that is made, to make sure that the cost doesn't run wild. -Or maybe a function that can be used after the recipe has been imported.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by fidoriel** [comment]
It would be very helpful if imperial units could also be converted into useful metric ones.
